### PR TITLE
Updating CNV VM import to RHV (2.6.1)

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1929174

### CODE REVIEW

File: modules/virt-importing-vm-cli.adoc [hunk 1]
```diff
@@ -169,8 +170,9 @@ EOF
 <9> If `storageMappings` are specified in both the `ResourceMapping` and the `VirtualMachineImport` CRs, the `VirtualMachineImport` CR takes precedence.
 <10> Specify the source storage domain.
 <11> Specify the target storage class.
-<12> Specify the source VM disk ID, for example, `8181ecc1-5db8-4193-9c92-3ddab3be7b05`. You can obtain the disk ID by entering `\https://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` in a web browser on the Manager machine and reviewing the VM details.
-<13> Specify the target storage class.
+<12> Specify `ReadWriteMany` or `ReadWriteOnce`.
```
Comment (apinnick) [inline modules/virt-importing-vm-cli.adoc:173]: @pkliczewski  Do the access settings require more explanation in this note?

Comment (istein1) [inline modules/virt-importing-vm-cli.adoc:173]: Please add: 
The value of `accessMode` may be one of `ReadWriteOnce`, `ReadWriteMany`, or `ReadOnlyMany`. 
If the access mode is not specified, the oVirt provider will attempt to determine the correct access mode.
based on the VM affinity (Migratable/Non migratable).
Reference:
https://github.com/kubevirt/vm-import-operator/commit/aa71489cf32526a8bc9aa408dd3551b18b56362d#diff-b0929dacd4f1fc192cd308c1a9db2e85154ea742ccec315518e659d7f6a3efc7R103

Comment (pkliczewski) [reply modules/virt-importing-vm-cli.adoc:173]: I would add that it is volume access mode.

Comment (apinnick) [reply modules/virt-importing-vm-cli.adoc:173]: Ilanit, the reference says that if no access mode is specified, the default is `ReadWriteOnce`.

Comment (istein1) [reply modules/virt-importing-vm-cli.adoc]: If no access mode is specified, the default is ReadWriteOnce - This refers to Import from VMware using VMIO, which we do not document.

Comment (apinnick) [reply modules/virt-importing-vm-cli.adoc]: @pkliczewski  If the source VM migration mode is "Allow manual migration only", what is the volume access mode?

File: modules/virt-importing-vm-cli.adoc [hunk 2]
```diff
@@ -130,47 +98,52 @@ spec:
...
-<9> If `storageMappings` are specified in both the `ResourceMapping` and the `VirtualMachineImport` CRs, the `VirtualMachineImport` CR takes precedence.
+<9> If storage mappings are specified in both the `ResourceMapping` and the `VirtualMachineImport` CRs, the `VirtualMachineImport` CR takes precedence.
 <10> Specify the source storage domain.
 <11> Specify the target storage class.
-<12> Specify the source VM disk ID, for example, `8181ecc1-5db8-4193-9c92-3ddab3be7b05`. You can obtain the disk ID by entering `\https://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` in a web browser on the Manager machine and reviewing the VM details.
-<13> Specify the target storage class.
+<12> Specify `ReadWriteOnce`, `ReadWriteMany`, or `ReadOnlyMany`. If no access mode is specified, {virt} determines the correct volume access mode based on the VM *Host* -> *Migration mode* setting:
+* *Allow manual and automatic migration*: `ReadWriteMany`
+* *Do not allow migration*: `ReadWriteOnce`
+* *Allow manual migration only*: `ReadOnlyMany`??
```
Comment (apinnick) [inline modules/virt-importing-vm-cli.adoc:144]: @pkliczewski  If the VM's migration mode is "Allow manual migration only", what is the volume access mode?

Comment (pkliczewski) [reply modules/virt-importing-vm-cli.adoc:143]: In order to enable vm migrations we need `ReadWriteMany`. VM migration is not possible when `ReadWriteOnce`.

Comment (apinnick) [reply modules/virt-importing-vm-cli.adoc:143]: @pkliczewski  This is about VM import, not VM migration. 

If the user imports a RHV VM with migration mode "Allow manual migration only" and **does not choose a volume access mode**, what access mode will oVirt choose?

Comment (pkliczewski) [reply modules/virt-importing-vm-cli.adoc:143]: When access mode is not overridden by the user it will be derived from specific disk setting in RHV.

Comment (apinnick) [reply modules/virt-importing-vm-cli.adoc:143]: @pkliczewski 
What is the relationship between the RHV disk settings and the access mode **if the access mode is not specified by the user**?

Comment (pkliczewski) [reply modules/virt-importing-vm-cli.adoc:143]: RHV disk and PVC for which we set access mode represent vm disk.

Comment (apinnick) [reply modules/virt-importing-vm-cli.adoc:143]: @pkliczewski 
What is the correlation between RHV disk and virt volume access mode? We want to tell the user what will happen, depending on what their VM settings are.

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**